pkg/network/tracer/event.go

```diff
@@ -376,6 +390,30 @@ func newIPRouteDest(source, dest util.Address, netns uint32) *ipRouteDest {
 	return d
 }
 
+func getPortType(p uint16) network.EphemeralPortType {
+	initfunc := func() {
+		procfsPath := "/proc"
+		if config.Datadog.IsSet("procfs_path") {
+			procfsPath = config.Datadog.GetString("procfs_path")
+		}
+
+		ephemeralIntPair = sysctl.NewIntPair(procfsPath, "net/ipv4/ip_local_port_range", time.Hour)
+	}
+	initEphemeralIntPair.Do(initfunc)
+
+	low, hi, err := ephemeralIntPair.Get()
+	if nil == err {
```

Comment:
nit: `err == nil`. We don't use yoda-style checks in Golang because it doesn't allow you to do assignment in control statement unless you use the special syntax `if a = ...; a != nil {`